src/CPISync/GossipNode.h

```diff
@@ -0,0 +1,29 @@
+/*
+*   Node class, where each node has a leveldb store and can communicate through CPISync.
+*   Zhe Deng Networking Inside of Kubernetes EC528 Spring 2020
+*/
+
+#pragma once
+#include "GossipUtilities.h"
+
+namespace niok
+{
+class GossipNode
+{
+public:
+//Metadata
+string name;
+vector <string> log;
+vector <string> neighbors;
+//Data
+DB* db = nullptr;
```

Comment:
I'll import the kvstore part code later

Reply:
Didn't get a chance to add the kvengine, so feel free to do that.